content/news/2024.md

```diff
@@ -53,6 +53,15 @@ More info...
 - bug fix #2 
 -->
 
+## June 2024
+
+### <span class="label label-news label-added">ADDED</span> Delete family by code
+
+Deleting a family was juts possible in the UI and we added an endpoint to allow for API deletion.
+This will also allow you to mass delete family with a script if needed.
```

Comment:
```suggestion
Deleting a family was only possible through the UI. Now we are adding a new endpoint to allow family deletion via the API.
```
```suggestion
Deleting a family was juts possible in the UI and we added an endpoint to allow for API deletion.
This will also allow you to mass delete family with a script if needed.
```

Peut-être à valider avec Chris mais pas sûre que ce soit utile de mettre le truc du mass delete? C'est très particulier à un client nan?